### PR TITLE
update game mechanism

### PR DESCRIPTION
**Pull request by tyeung2017**
Added randomising balloon colour and breakpoint

Game mechanics is kind of complete

As for game logic, it will be decided later.

PS. I know I mixed up the branches from the CSS one, it was approved anyway.

### CODE REVIEW

**Review by KatiaAsh — APPROVED** [review]
:100:

File: game.js [hunk 1]
```diff
@@ -44,7 +47,31 @@ function balloonKnot(x, y){
...
+      breakPoint = randomizer(10) + 110;
+      // console.log(breakPoint);
+      break;
+    case 'red':
+      breakPoint = randomizer(70) + 50;
+      // console.log(breakPoint);
+      break;
+    default:
+      breakPoint = randomizer(20) + 50;
+      // console.log(breakPoint);
```
Comment (Karyum) [inline game.js:68]: commented out logs

File: game.js [hunk 2]
```diff
@@ -44,7 +47,31 @@ function balloonKnot(x, y){
...
+  return Math.floor(Math.random() * num);
+}
+
+function colorPick(){ //yeah, we know having side effect to change global variable is not ideal
+  var pickColor = randomizer(3);
+  var result = pickColor===3?2:pickColor;
+  switch (colorChart[result]) {
+    case 'gold':
+      breakPoint = randomizer(10) + 110;
+      // console.log(breakPoint);
```
Comment (Karyum) [inline game.js:60]: here

File: game.html [hunk 3]
```diff
@@ -3,16 +3,29 @@
...
 </head>
 
 <body>
-  <canvas id="canvas" width="500" height="500"></canvas>
-  <div id="counter"></div>
-  <div id="collect-counter"></div>
-  <div id="balloon-counter"></div>
-  <button id="pump">Pump</button>
-  <button id="collect">Collect</button>
+  <div class="game">
```
Comment (Karyum) [inline game.html:10]: do you need this div it's not used anywhere?

**Comment by Wyrd00** [comment]
sorry, I didn't mean to merge this - early morning mind fog.
mario, please let us know if there is anything wrong with the codes?

**Pull request merged**